src/gpu/generic/sycl/ref_batch_normalization.cpp

```diff
@@ -84,9 +84,13 @@ status_t ref_batch_normalization_fwd_t::init(impl::engine_t *engine) {
                 = ::sycl::get_kernel_id<batch_normalization_fwd_kernel_vec_t>();
         CHECK(create_kernel(engine, kid, &kernel_));
     } else {
+        // Enabling the IEEE div compliant implementation
+        setenv("SYCL_PROGRAM_COMPILE_OPTIONS",
```

Comment:
I believe this is not thread-safe to use setenv (e.g. another concurrent call to the same primitive might unset it after this one sets it).